src/pages/GamePage/components/CharacterFigure.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  halfOrc, elf, dwarf, human,
+} from '../../../assets';
```

Comment:
`import {
  halfOrc, 
  elf, 
  dwarf,
  human,
} from '../../../assets';`

... Is a better convention